src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -214,29 +227,44 @@ private bool ValidateRootBuildOptions()
         }
 
         /// <summary>
-        ///     Obtain the root build.
+        ///     Obtain the root builds to start with.
         /// </summary>
-        /// <returns>Root build to start with.</returns>
-        private async Task<Build> GetRootBuildAsync()
+        /// <returns>Root builds to start with, or null if a root build could not be found.</returns>
```

Comment:
Should this include "or command line options are invalid"?

What's the difference between null and an empty collection?

Seems like a strange API. It could *also* throw an exception, but the comment describing the exception appears incorrect....

Reply:
Empty collection would also be invalid.

Reply:
Weird, but fine leaving it at that. Wasn't introduced by this PR.